hyperspy/_signals/signal2d.py

```diff
@@ -778,6 +785,126 @@ def add_ramp(self, ramp_x, ramp_y, offset=0):
         ramp += ramp_y * yy
         self.data += ramp
 
+    def find_peaks(self, method='local_max', interactive=True,
+                   current_index=False, show_progressbar=None,
+                   parallel=None, display=True, toolkit=None, **kwargs):
```

Comment:
`parallel=None, max_workers=None, ...` for consistency with other methods like `s.align()`. See notes below for the docstring as well.